sch_fq.c

```diff
@@ -543,30 +527,25 @@ static struct sk_buff *fq_dequeue(struct Qdisc *sch) {
 
   int rValue = valuePresentInArray(f->socket_hash, pFlowid, lengthOfarray);
```

Comment:
could we rename variables like "lVal" or "rValue" with something more descriptive?